doto-frontend/src/components/pages/Settings/__snapshots__/SettingsPage.test.js.snap

```diff
@@ -16,23 +16,22 @@ exports[`<SettingsPage /> component being rendered Make sure render matches snap
       >
         Settings
       </h1>
-      <ul
-        className="IconList"
+      <div
+        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end"
```

Comment:
This classname seems a little odd, I can't seem to find out where `IconList` changes to what is here.

Reply:
Yeah thats from a previous change where the snapshot wasnt updated.